lib/charms/mongodb/v0/mongodb.py

```diff
@@ -30,11 +30,13 @@
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 7
+LIBPATCH = 8
 
 # path to store mongodb ketFile
 logger = logging.getLogger(__name__)
 
+MONGODB_PORT = 27017
```

Comment:
up to @Mehdi-Bendriss, maybe it can be `Config.MONGODB_PORT` like in `lib/charms/mongodb/v1/mongos.py`
```suggestion
from config import Config
```

Reply:
sounds good to me, but this will have to be done on the MongoDB side, since this charm does not own the lib :)